### ISSUE TITLE
Bug Found in DecisionTrees.Learning.ID3Learning

### ISSUE THREAD

**Issue by lwqaitd** [accord-net/framework#215]
I found a bug in Accord.MachineLearning.DecisionTrees.Learning.ID3Learning
the property MaxHeight
```
set
{
               if (maxHeight <= 0)
                {
                    throw new ArgumentOutOfRangeException("value",
                        "The height must be greater than zero.");
                }
                maxHeight = value;
}

```
the first maxHeight should be 'value'.

**Comment by cesarsouza** [comment]
Thanks!

**Comment by volend** [comment]
+1

**Comment by cesarsouza** [comment]
Fixed on release 3.2.
